server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailHeader.scala

```diff
@@ -75,12 +76,35 @@ object GroupedAddressesHeaderValue extends EmailHeaderValue {
   }
 }
 
+object MessageIdsHeaderValue {
+  def from(field: Field): MessageIdsHeaderValue = {
+    val messageIds: List[HeaderMessageId] = format(field.getBody)
+      .split(" ")
```

Comment:
https://alvinalexander.com/scala/how-to-split-csv-strings-in-scala-cookbook/

```
About that split method ...

The split method is overloaded, with some versions of the method coming from the Java String class and some coming from the Scala StringLike class. For instance, if you call split with a Char argument instead of a String argument, you’re using the split method from StringLike:

// split with a String argument
scala> "hello world".split(" ")
res0: Array[java.lang.String] = Array(hello, world)

// split with a Char argument
scala> "hello world".split(' ')
res1: Array[String] = Array(hello, world)
```

So let's avoid the regex version of it, use the char version of it! (I don't like Regexes!)